cleanlab_studio/utils/tlm_lite.py

```diff
@@ -239,18 +245,11 @@ def _try_batch_score(
 
         assert len(prompt) == len(score_response)
 
-        if all(score is None or isinstance(score, dict) for score in score_response):
-            score_response = cast(List[Optional[TLMScore]], score_response)
+        if all(isinstance(score, dict) for score in score_response):
             return [
                 {"response": res, **score} if score else None
                 for res, score in zip(response, score_response)
             ]
-        elif all(score is None or isinstance(score, (float, int)) for score in score_response):
-            score_response = cast(List[Optional[float]], score_response)
-            return [
-                {"response": res, "trustworthiness_score": score} if score else None
-                for res, score in zip(response, score_response)
-            ]
         else:
             raise ValueError(f"score_response has invalid type")
```

Comment:
Same here re else statement

Reply:
this is also just trying to pass the mypy check